api/services/registration_service.py

```diff
@@ -8,7 +8,7 @@ class RegistrationService:
     # Create a registartion
     def create_registration(self, data):
         system_phone_data = models.SystemPhone.query.filter_by(
-            phone=data["to_number"]
+            phone=data["to_number"][-10:]

```

Comment:
Is it fine to check the last 10 digits, as previously we were not doing this?

Reply:
No, @vivek-kumar-pandey I forgot to remove this I was just testing all the changes. So, added it. Removing now.